raiden/api/v1/encoding.py

```diff
@@ -273,3 +271,16 @@ class ConnectionsConnectSchema(BaseSchema):
     class Meta:
         strict = True
         decoding_class = dict
+
+
+class ConnectionsLeaveSchema(BaseSchema):
+    only_receiving_channels = fields.Boolean(
+        required=False,
+        default=True,
+        missing=True,
+        location='json'
```

Comment:
I don't think `marshmallow.fields.*` can receive a `location` keyword. They go only in the `webargs.use_kwargs` decorator, and here, are silently ignored.

Reply:
Hmmm then we got it in other places wrong too. By looking at [the docs](http://marshmallow.readthedocs.io/en/latest/api_reference.html#marshmallow.fields.Field) it seems you are correct.